### PR TITLE
Merge doc prod updates into master

### PR DESCRIPTION
**Pull request by LiamConnors**
This merges recent updates to the docs back to master

### CODE REVIEW

File: doc/python/dendrogram.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ from scipy.spatial.distance import pdist, squareform
 # get data
 data = np.genfromtxt("http://files.figshare.com/2133304/ExpRawData_E_TABM_84_A_AFFY_44.tab",
                      names=True,usecols=tuple(range(1,30)),dtype=float, delimiter="\t")
-data_array = data.view((np.float, len(data.dtype.names)))
+data_array = data.view((float, len(data.dtype.names)))
```
Comment (archmoj) [inline doc/python/dendrogram.md:95]: Do we need this change?

Comment (LiamConnors) [reply doc/python/dendrogram.md:95]: Yes, `np.float` has been removed in recent versions of Pandas
Here's the original PR that add that to the docs https://github.com/plotly/plotly.py/pull/4364/files

Comment (archmoj) [reply doc/python/dendrogram.md:95]: Thank you!

**Review by archmoj — APPROVED** [review]
Nicely done.
:dancer:

**Pull request merged**